src/style.css

```
@@ -0,0 +1,90 @@
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-flow: column nowrap;
+  width: 200px;
+  height: 403px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: Roboto;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+  padding: 14px 14px 15px;
+}
+
+.product-image {
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 0;
+}
+
+.product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 45px auto 0;
```

This is a flex element, try to centered using justify-content in a parent.

---

I've tried, then it moves left for some reason

---

I meant you don't need use `auto` in `margin`
```suggestion
  margin: 45px 0 0;
```